src/client/index.tsx

```diff
@@ -122,9 +123,11 @@ const setupUserSession = async (userManager: UserManager, store: Store) => {
 const renderApp = () => {
   return ReactDOM.render(
     <Provider store={store}>
-      <ConnectedRouter history={history}>
-        <App />
-      </ConnectedRouter>
+      <NexusContext.Provider value={nexus}>
```

Comment:
I feel really uncomfortable doing this, especially when there is already a redux action setup for files. I'm ok to sit down together on this one whenever you're free